terminal/README.md

```diff
@@ -0,0 +1,4 @@
+# GoTTY - Share your terminal as a web application
+
+> GoTTY is a simple command line tool that turns your CLI tools into web applications.
+## [Repository](https://github.com/yudai/gotty)
```

Comment:
Repo is here:
https://github.com/okteto/terminal

Reply:
thanks for pointing to the right link.